src/core/sort.cc

```diff
@@ -1469,10 +1469,8 @@ py::oobj py::Frame::sort(const PKArgs& args) {
   dt::expr::EvalContext ctx(dt);
 
   if (args.num_vararg_args() == 0) {
-    py::otuple all_cols(dt->ncols());
-    for (size_t i = 0; i < dt->ncols(); ++i) {
-      all_cols.set(i, py::oint(i));
-    }
+    size_t n_cols = 0;
+    py::otuple all_cols(n_cols);
```

Comment:
I guess now a better name for this variable is `no_cols`, so you can just do `py::otuple no_cols(0);` here.

Reply:
Done